src/Select.js

```diff
@@ -498,6 +501,17 @@ const Select = React.createClass({
 					event.stopPropagation();
 				}
 			break;
+			case 32: // space
+				if (!this.props.searchable) {
+					event.preventDefault();
+				}
+				if (!this.state.isOpen) {
+					this.focusNextOption();
+					return;
+				}
+				event.stopPropagation();
+				this.selectFocusedOption();
```

Comment:
Hey guys, sorry to drudge up an old PR discussion, but I'm noticing some quirky behavior since this code.  Bisecting for bug #2156 led me to PR #2092 and therefore here.

It would seem this didn't intentionally cause the functionality I'm noticing on multiselect (selecting options on space or creating options on space for `Creatable`), so I'm trying to figure out how I might fix this bug while keeping the intentions of this change.  

What use case are we trying to add here?  As an example of a use case we don't want (I'm assuming) is if you're on the example page in the first example of states and click "United States," then start typing to New York, when you hit space after `New` it will select `New Hampshire` instead of continuing to filter down.

Tagging @gwyneplaine as the merger.

Reply:
Hey @buob  thanks for catching this, the intention was to mimic native select behaviour, to enable selection of a focused option on SPACE. this is definitely an unintended side-effect. 

Let me have a think about how to best solve for both these cases (it may very well be removing this behaviour when props.searchable is true). Happy to hear your thoughts on this as well

Reply:
I went hunting, and this seems to be the most relevant. I'm still seeing a space to select items here, but my search is supposed to support space characters for my options.

Was there ever an option added to disable the space select on a multiselect search?